store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/StoreWriteRequest.kt

```diff
@@ -0,0 +1,21 @@
+package org.mobilenativefoundation.store.store5
+
+import kotlinx.datetime.Clock
+import org.mobilenativefoundation.store.store5.impl.OnStoreWriteCompletion
+import org.mobilenativefoundation.store.store5.impl.RealStoreWriteRequest
+
+interface StoreWriteRequest<Key : Any, CommonRepresentation : Any, NetworkWriteResponse : Any> {
```

Comment:
I think eventually we should collapse to

Store.Request.Write and Store.Request.Read 

same with the response